meeting-minutes/2024-11-20.md

```diff
@@ -0,0 +1,165 @@
+| Tobias     | Limmer     | Siemens                                                     | Member                    | NO      |
+
+Quorum was reached (13 voting members present)
+
+## 1.3 Procedures for this meeting (Moderator)
+
+## 1.4 Approval of agenda
+
+* Roll Call  
+* Updates:  
+  * Happy (belated) birthday OpenEOX!
+  * CSAF Community Days Dec 12th and 13th
+    * <https://oasis-open.github.io/csaf-documentation/communitydays/>
+* CVE and CPE discussion (Feng and Przemyslaw)
+* Approval of previous meeting minutes (motions carried out per e-mail motions)  
+* Review of outstanding issues and pull requests marked for TC discussion: https://github.com/oasis-tcs/openeox  
+  * [Meeting minutes August and September #50](https://github.com/oasis-tcs/openeox/issues/50)
+  * [CISA proposed definition for "security support" as part of NCSIP Initiative 3.3.2 #51](https://github.com/oasis-tcs/openeox/issues/51)
+  * [End-of-Software Maintenance #29](https://github.com/oasis-tcs/openeox/issues/29) 
+    * based onn last month's discussion for [Using "End" word in the lifecycle definitions #38](https://github.com/oasis-tcs/openeox/issues/38)
+  * [End-of-Security Vulnerability Support #32](https://github.com/oasis-tcs/openeox/issues/32)
+  * [Suggested new optional field "successor" #3](https://github.com/oasis-tcs/openeox/issues/3)
+* Next steps  
+
+Agenda aggreed.
+
+## 1.5 Approval of previous minutes (Moderator)
+
+None (motions carried out already per e-mail motions).
+
+## 1.6 Review of action items and resolutions (Secretary Stefan)
+
+None
+
+## 1.7 Identification of TC voting members (Secretary)
+
+- the roster is out of sync so no statements on member status changes possible during meeting
+
+### 1.7.1 Prospective voting members attending their first meeting
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+### 1.7.5 Members who have declared a leave of absence
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Secretary)
+
+- Scheduled Teleconferences (Wednesday at 09:00 PT / 12:00 ET / 18:00 CET / **17:00** UTC for 1 hour)
+
+    ```
+    December 18, 2024
+    ```
+  - regrets from Stefan Hagen
+
+# 3. Discussion
+
+- [Meeting minutes August and September #50](https://github.com/oasis-tcs/openeox/issues/50)
+  - Stefan: Please provide the recordings to fix the roster
+  - Justin: Will see that the recordings will be sent to Stefan for extracting
+- [CISA proposed definition for "security support" as part of NCSIP Initiative 3.3.2 #51](https://github.com/oasis-tcs/openeox/issues/51)
+  - Justin: Motivates the issue and invites everyone to discuss the proposed definition in the issue
+  - Langley: We are not responding to security events or security incidents but not to security risks 
```

Comment:
I assume one of these "not"s is wrong?

Reply:
@lfrancke good catch. As noted in #51, should read as:
"**Not** responding to security risk, but to security events/incidents."

Reply:
Resolved. Thanks for the catch @lfrancke